fitnessllm_dataplatform/batch_handler.py

```
@@ -43,7 +91,9 @@ def process_user(self, user_id: str, data_source: str = "strava") -> None:
             raise
 
     @beartype
-    def process_all_users(self, data_source: str = "strava") -> None:
+    def process_all_users(
+        self, data_source: str = FitnessLLMDataSource.STRAVA.value
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)